docs/docs/welcome.mdx

```diff
@@ -31,30 +31,30 @@ import Demo from '../examples/Demo.js';
 [![Star on GitHub][github-star-badge]][github-star]
 [![Tweet][twitter-badge]][twitter]
 
-Reactjs-popup is a simple react popup component that help you create simple and complex Modals, tooltips and Menus for your next React App.
+`reactjs-popup` is a simple React popup component that help you create simple and complex Modals, Tooltips and Menus for your next React App.
```

Comment:
I thought that `reactjs-popup` stands out more and is a bit more visible than "Reactjs-popup"